device-types/Supermicro/SYS-5018D-MTF.yaml

```diff
@@ -0,0 +1,26 @@
+---
+manufacturer: Supermicro
+model: SYS-5018D-MTF
```

Comment:
The model appears to actually be `SuperServer 5018D-MTF` according to their website